pkg/controller/routetable/routetable_controller.go

```diff
@@ -160,12 +161,12 @@ func (r *ReconcileRouteTable) Reconcile(request reconcile.Request) (reconcile.Re
 				if aerr, ok := ierr.(awserr.Error); ok {
 					switch aerr.Code() {
 					default:
-						fmt.Println(aerr.Error())
+						//fmt.Println(aerr.Error())
```

Comment:
Why did we comment out all of the things that only show up if there are system issues?

Reply:
I had received some feedback a while back on not having anything print to the screen- I had commented them out, figuring I'd setup logging when I go back and refactor. I did go ahead and change those to log the events in the resource for this pull request.